.github/workflows/release.yml

```diff
@@ -335,7 +335,6 @@ jobs:
           docker run --rm test-${{ matrix.distro }}
 
   smoke-tests:
-    if: startsWith(github.ref, 'refs/tags') && !github.event.pull_request.draft
```

Comment:
In case you are unaware, this gate was added at the request of the squads in the interests of runtime. I cannot remember who raised it but I believe Leo made the commit. Also, the test that you have added below is not affected by this gate anyway.

Personally, I think it is fine to run it as the plugin compiler can break even if none of its code is touched in a PR.

Reply:
Also, the plugin compiler tests depend on public artefacts so cannot run for every PR. Public attracts are pushed only when a tag is pushed.

Reply:
Reverted this due to the tests requiring a tag.